.nupic_modules

```diff
@@ -1,3 +1,3 @@
 # Default nupic.core dependencies (override in optional .nupic_config)
 NUPIC_CORE_REMOTE = 'git://github.com/numenta/nupic.core.git'
-NUPIC_CORE_COMMITISH = 'f55119a2f255fa4961b9c44ecb5261076bb4db0a'
\ No newline at end of file
+NUPIC_CORE_COMMITISH = 'cb55ebd9633923040176db0c041945f190d69800'
```

Comment:
cb55ebd9633923040176db0c041945f190d69800  exists in nupic.core, so I am assuming that it's the one that you intended.